tests/AutoReview/ProjectCodeTest.php

```diff
@@ -526,34 +525,6 @@ public function testAllCodeContainSingleClassy(string $className): void
         static::assertNull($tokens->getNextNonWhitespace($classyEndIndex), sprintf('File "%s" should only contains a single classy.', $file));
     }
 
-    /**
-     * @dataProvider provideSrcClassCases
-     */
-    public function testThereIsNoTriggerErrorUsedDirectly(string $className): void
```

Comment:
The added deprecation in this PR is not intended for end-users but for library authors. I don't know how to keep this test and tell it which `trigger_error()` calls to allow reliably.